tools/workload/schema/update_bank.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
```

Comment:
Please add a document to describe the detailed scenario of this test case. Including 
1. wide/narrow table?
2. update mode
3. hotspot
4. etc

Reply:
added, PTAL.